tidb-cloud/tidb-cloud-encrypt-cmek.md

```diff
@@ -0,0 +1,125 @@
+- Currently, for a specific project, you can only enable CMEK for one AWS region. Once you have configured it, you cannot create clusters in other regions within the same project.
+
+## Enable CMEK
+
+If you want to encrypt your data using the KMS owned by your account, take the following steps.
+
+### Step 1. Provision KMS and IAM in your cloud provider
+
+1. Create your CMEK Key on the AWS Key Management Service (KMS) console. Copy the KMS Key ARN. To learn how to create a key, see [Creating Keys](http://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html#create-symmetric-cmk) in the AWS documentation.
+
+2. Create an IAM role and configure the role's access policy to CMEK.
+
+    To make the TiDB cluster function as expected, you need to grant certain permissions to the TiDB account for accessing KMS. Note that this feature is in the development stage and the policy requirement could be subject to change as future features might require more permissions. Here’re the required permissions for supported features at present:
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Id": "cmek-policy",
+        "Statement": [
+            // EBS-related policy
+            {
+                "Sid": "Allow access through EBS for all principals in the account that are authorized to use EBS",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "*"
+                },
+                "Action": [
+                    "kms:Encrypt",
+                    "kms:Decrypt",
+                    "kms:ReEncrypt*",
+                    "kms:GenerateDataKey*",
+                    "kms:CreateGrant",
+                    "kms:DescribeKey"
+                ],
+                "Resource": "*",
+                "Condition": {
+                    "StringEquals": {
+                        "kms:CallerAccount": "<pingcap-account>",
+                        "kms:ViaService": "ec2.us-west-2.amazonaws.com"
+                    }
+                }
+            },
+            // S3-related policy
+            {
+                "Sid": "Allow TiDB cloud role to use KMS to store encrypted backup to S3",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::<pingcap-account>:root"
+                },
+                "Action": [
+                    "kms:Decrypt",
+                    "kms:GenerateDataKey"
+                ],
+                "Resource": "*"
+            },
+            ... // user's own admin access to KMS
+        ]
+    }
+    ```
+
+> **Note:**
+>
+> - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
```

Comment:
```suggestion
> - `<pingcap-account>` is the account where your clusters run in. If you do not know the account, contact [support](/tidb-cloud/tidb-cloud-support.md).
> - `<region>` is the region where you want to create your cluster.
```

Reply:
> If you do not want to specify a region, replace `<region>` with a wildcard `*`.
If you do not want to specify a single region, you can create a `StringLike` clause and use a wildcard `*` for all regions.

Notice `*` in `StringEquals` doesn't work, it need to be in a `StringLike` block.

Reply:
can you give an example for `StringLike` clause?